src/traces/sunburst/helpers.js

```diff
@@ -13,15 +13,16 @@ var Color = require('../../components/color');
 var setCursor = require('../../lib/setcursor');
 var pieHelpers = require('../pie/helpers');
 
-function hasLabel(label) {
-    return label || label === 0;
+function labelStr(label) {
+    return (label || label === 0) ? label + '' : '';
```

Comment:
```js
(label || label === 0) ? String(label) : '';
```

would be better here.

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/4185/commits/4d790a1af924968763d0cba30fb9ae6b047ec044.